### DISCUSSION TITLE
Have to reinstall app after every close down

### DISCUSSION THREAD

**Discussion by r1ckp1nk** [Rem0o/FanControl.Releases#829] [General]
Hi everytime i boot my pc up i seem to have to re-run the .exe folder and do a fresh install. Also when i load my saved configuration, it doesnt load up my curve graphs i assigned to it.

This is possibly user error but if i can get some assistance that would be great.

Its a great app and has improved my operating temps over 20% already, just wish i didnt repeatedly have to reinstall and calibrate it everytime.

**Comment by AtA3301** [comment]
It's just a folder, you don't really install it. Where have you put the FanControl folder? The only "installing" that's done is the start-on-boot part that runs it - is that what is missing perhaps? Try adding the FanControl.exe to startup manually? Make sure you save the settings when you set up the fans, and make a copy of the json file to be safe.

**Reply by Rem0o** [reply]
^ This, There is no "installing". It's just an executable.

**Comment by UrmasK222** [comment]
I had the same issue when I had FanControl in "Downloads" folder and OneDrive was actively syncing. So I just created & moved it in a folder that's not related to my user profile. For me it's "C:\Tools\FanControl" 

You could also try with fresh new profiles just to see if that helps.

Try starting from scratch. Something like this could help

1. Backup FC profiles manually from it's root directory to another directory
2. Deselect "Start with Windows" from FanControl software
3. Find and delete all FanControl folders
4. Restart Windows
5. Download newest version of FanControl & extract it to your new independent folder
6. Copy profiles to that folder
7. Select "Start with Windows"
8. Try now

1: 
![image](https://user-images.githubusercontent.com/45017739/176834701-7221cab2-3f79-4398-9b33-f081ab2bf4f2.png)


2: 
![image](https://user-images.githubusercontent.com/45017739/176834541-bbff8617-cf84-4101-a3e2-97b18d6969b3.png)

br

**Comment by r1ckp1nk** [comment]
All good now thanks. Sorted it a while back
